integreat_cms/cms/views/poi_categories/poi_category_form_view.py

```diff
@@ -118,11 +147,12 @@ def post(self, request, *args, **kwargs):
         :return: The redirect
         :rtype: ~django.http.HttpResponseRedirect
         """
-        form = self.get_form()
+        self.poi_category_form = self.get_poi_category_form()
```

Comment:
```suggestion
        form = self.get_form()
```
This can be reverted